openfisca_aotearoa/tests/social_security/accommodation_supplement/2018/exclusion.yaml

```diff
@@ -87,27 +92,31 @@
 
 - name: >
     Social Security Act 2018 -
-    65 Accommodation Supplement -
-    66 Social housing exclusion
-    (c) Residential care
+    Subpart 10 Accommodation Supplement -
+    67  Other funding exclusion -
+        (c) Residential care
   period: 2018-11-26
   input:
     accommodation_type:
       2022-W01: [residential_care, under_the_moon]
```

Comment:
What is the purpose of `under_the_moon`? It's not a valid AccommodationType value, note the other comment regarding `under_the_sun`. I'm not entirely sure why OpenFisca accepts this.

Reply:
Hey! Yes, these are arbitrary values.